src/components/TodoItem.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+function TodoItem(props) {
+  if (!props.display) {
+    return null;
+  } else if (props.textMatch) {
+    return (
+      <article className={props.isCompleted ? "card completed" : "card"}>
```

Comment:
how does this article differs from the one on line 33 https://github.com/andrewgaryga/Corporate-task-manager--React-/pull/1/files#diff-1d289629e0efd3fbf40e4fbe1cca99c7R33 ?

Reply:
First 'if' for display todo item with highlighted text from search:

`{props.textMatchStart}<span className="highlighted">{props.textMatch}</span>{props.textMatchEnd}`

else if for displaying unhighlighted text:

`{props.message}`

I dunno how to make it inside one render.